index.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Learning HTML and CSS</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+  	<div class = "title">
+  		<h1>Git</h1>
+		<p>This webpage will answer these 3 questions.</p>
+		<p>What is Git?</p>
+		<p>Why do developers need Git?</p>
+		<p>What is a branch in Git?</p>
+	</div>
+		
+	<main>
+		<div class="parent">
+		    <div> <img src = "git.png" alt = "git logo" class = "center">
+	    	<article>
+			<h3>What is Git?</h3>
+				<p>Git basically takes a picture of what all your files look like at that moment and stores a reference to that snapshot.</p>
+			<nav><a href = "https://git-scm.com/book/en/v2/Getting-Started-What-is-Git%3F" target = "_blank">Read more</a></nav>
```

Comment:
A nav is a grouping element. If you have just one link, do you need to group it?

Reply:
Hi

I have made the changes.